traffic_ops/traffic_ops_golang/server/servers_server_capability.go

```diff
@@ -441,3 +442,87 @@ func getDSTenantIDsByIDs(tx *sqlx.Tx, dsIDs []int64) ([]DSTenant, error) {
 
 	return dsTenantIDs, nil
 }
+
+// AssignMultipleServerCapabilities helps assign multiple server capabilities to a given server.
+func AssignMultipleServerCapabilities(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, []string{"id"}, []string{"id"})
```

Comment:
This required `id` route parameter is undocumented - and it doesn't appear to be used. The payload includes a server ID, and that appears to be what's used throughout the handler to identify the server being acted upon. So I think this parameter can be removed (and also removed from the pattern in the route definition in `routes.go`)

Reply:
You are right to remove it from here.. again had a different train of thought when I was coding the logic and this was residue to that train of thought.. but remove it from routes.. hmm... i see why are you suggesting.. ok.. updating the routes.